src/pages/library/Code.js

```diff
@@ -96,7 +96,7 @@ const Code = () => {
       withBusyState(setLoading)
     )(async () => {
       const [newFeaturedList, newMethods] = await Promise.all([
-        fetch(`${getConfig().firecloudBucketRoot}/featured-methods.json`, { signal }).then(res => res.json()),
+        Ajax(signal).FirecloudBucket.getFeaturedMethods(),
```

Comment:
I am passing signal to this one since Methods call right below is already using signal.

Reply:
What is `signal` for?

Reply:
It allows the operation to be cancelled if you navigate away from the page. So typically you use it for GET operations, but definitely not for an operation that should complete even if you navigate away (such as a PUT/POST to update server state).

Reply:
Most of these calls weren't using `signal`, and I didn't attempt to decide if they actually should be except for the 2 cases where it was very obvious because there were 2 different pieces of information being obtained from the server, and the other was already using a cancellation signal.

Reply:
Oh, and actually this call was already using `signal`… I meant to make the comment below on the usage in the Workflows file.